src/components/Dashboard.tsx

```diff
@@ -0,0 +1,34 @@
+import { Session } from "@supabase/supabase-js";
+import ProjectDropdown from "./ProjectDropdown";
+import WordCountForm from "./WordCountForm";
+import WordCountProgressBar from "./WordCountProgressBar";
+import { useEffect } from "react";
+import { supabase } from "../supabase";
+
+export interface Props {
+  session: Session;
+}
+
+export default function Dashboard({ session }: Props) {
```

Comment:
Can we call this `ExtensionDashboard` or something? Since there will be a fullscreen dashboard later

Reply:
Couldn't we reuse this for fullscreen?